codespell_lib/data/dictionary.txt

```diff
@@ -25706,7 +25754,7 @@ wiew->view
 wigdet->widget
 wigdets->widgets
 wighed->weighed, wicked,
-wight->weight, white, right, wight,
+wight->weight, white, right, write, wight, disabled due to being a person's name
```

Comment:
I'd rather not disable for names (maybe unless they are very common, but this one doesn't seem all that common)

Reply:
removed disabled status 6799b3a